plugins/MantisGraph/MantisGraph.php

```diff
@@ -227,7 +227,7 @@ private function get_url_with_filter( $p_page ) {
 	 */
 	function summary_menu() {
 		$t_menu_items[] = '<a href="'
-			. $this->get_url_with_filter( 'developer_graph.php' )
+			. $this->get_url_with_filter( 'project_graph.php' )
```

Comment:
Changing the default graph is fine, but it needs to be reflected in all graph scripts
```php
print_summary_menu( 'project_graph.php', $t_filter );
```
Otherwise the *Graphs* tab will no longer be marked as active.